internal/admin/handle_export_importers.go

```diff
@@ -0,0 +1,157 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package admin
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+	"github.com/google/exposure-notifications-server/internal/exportimport/database"
+	"github.com/google/exposure-notifications-server/internal/exportimport/model"
+)
+
+// HandleExportImportersSave handles the create/update actions for export
+// importers.
+func (s *Server) HandleExportImportersSave() func(c *gin.Context) {
+	return func(c *gin.Context) {
+		var form exportImporterFormData
+		if err := c.Bind(&form); err != nil {
+			ErrorPage(c, err.Error())
+			return
+		}
+
+		ctx := c.Request.Context()
+		m := TemplateMap{}
+
+		db := database.New(s.env.Database())
+		model := new(model.ExportImport)
+
+		idRaw := c.Param("id")
+		if idRaw != "" && idRaw != "0" {
+			id, err := strconv.ParseInt(idRaw, 10, 64)
+			if err != nil {
+				ErrorPage(c, "failed to to parse `id` param.")
+				return
+			}
+
+			model, err = db.GetConfig(ctx, id)
+			if err != nil {
+				ErrorPage(c, fmt.Sprintf("failed to load export importer config: %s", err))
+				return
+			}
+		}
+
+		if err := form.BuildExportImporterModel(model); err != nil {
+			ErrorPage(c, fmt.Sprintf("failed to build export importer config: %s", err))
+			return
+		}
+
+		fn := db.AddConfig
+		if model.ID != 0 {
+			fn = db.UpdateConfig
+		}
+
+		if err := fn(ctx, model); err != nil {
+			ErrorPage(c, fmt.Sprintf("failed to write export importer config: %s", err))
+			return
+		}
+
+		m.AddSuccess("Successfully updated export importer config!")
+		m["model"] = model
+		c.HTML(http.StatusOK, "export-importer", m)
+		c.Abort()
+	}
+}
+
+// HandleExportImportersShow handles the create/update actions for export
+// importers.
+func (s *Server) HandleExportImportersShow() func(c *gin.Context) {
+	return func(c *gin.Context) {
+		ctx := c.Request.Context()
+
+		db := database.New(s.env.Database())
+		model := new(model.ExportImport)
+
+		if idRaw := c.Param("id"); idRaw != "" && idRaw != "0" {
+			id, err := strconv.ParseInt(idRaw, 10, 64)
+			if err != nil {
+				ErrorPage(c, fmt.Sprintf("Failed to parse `id` param: %s", err))
+				return
+			}
+
+			model, err = db.GetConfig(ctx, id)
+			if err != nil {
+				ErrorPage(c, fmt.Sprintf("Failed to load export importer config: %s", err))
+				return
+			}
+		}
```

Comment:
This code seems duplicated also in `(*Server).HandleExportImportersSave` perhaps this helper
```go
func modelFromContext(c *gin.Context, db *model.ExportImportDB) (*model.ExportImport, erorr) {
    idRaw := strings.TrimSpace(c.Param("id"))
    if idRaw == "" || idRaw == "0" {
         // Just return a blank model.
         return new(model.ExportImport), nil
    }
   
    id, err := strconv.ParseInt(idRaw, 10, 64)
    if err != nil {
         return nil, fmt.Errorf("Failed to parse `id` param: %w", err)
    }

    model, err := db.GetConfig(ctx, id)
    if err != nil {
        err = fmt.Errorf("Failed to load export importer config: %w", err)
    }
    return model, err
}
```

Reply:
This PR is just moving existing code around. To reduce the size of a diff and to reduce the chance of introducing a bug, I'm only moving code, no refactors or other changes in this PR.